cmd/options/options.go

```diff
@@ -87,6 +87,9 @@ type NodeProblemDetectorOptions struct {
 
 	// NodeName is the node name used to communicate with Kubernetes ApiServer.
 	NodeName string
+
+	// UnderTest indicates that the options will bypass certain checks.
+	UnderTest bool
```

Comment:
What does this mean? I don't quite get it.

Reply:
I created a nullexporter in the test instead of adding this.